### ISSUE TITLE
MouseButton is not exported from '@h5web/lib'.

### ISSUE THREAD

**Issue by vallsv** [silx-kit/h5web#1141]
I am trying to use `Pan` with middle mouse button but it sounds like is is not exported.

```
import {
  MouseButton,
} from '@h5web/lib';
```

```
Attempted import error: 'MouseButton' is not exported from '@h5web/lib'.
```

Is it how it is supposed to be used?

**Comment by loichuder** [comment]
> Is it how it is supposed to be used?

Yes ! Sorry, oversight from us. 

I will export it and make a new release.

**Comment by loichuder** [comment]
Fixed in 5.0.1

**Comment by vallsv** [comment]
thanks a lot.
